docs/News.md

```diff
@@ -16,6 +16,33 @@
     
 *   **Gene Tables Update:** Updated tables of genes (main and alias), based on [Oct 1, 2023 HGNC release](http://ftp.ebi.ac.uk/pub/databases/genenames/hgnc/archive/monthly/tsv/). See seedDB release notes [here](https://github.com/cBioPortal/datahub/tree/master/seedDB#latest-seed-database-schema-2131) for details.
 
+## Dec 5, 2023
+* **New feature**: Filter by data types on the homepage:
+    ![image](https://github.com/cBioPortal/cbioportal/assets/1334004/cf6d5486-bfd7-4f69-add0-442dbb4d354a)
```

Comment:
I'm not sure if this is specific to how I'm previewing it, but the image is showing up next to the text rather than under, which makes the vertical spacing funky. Can you make the image go under the text?